c++/src/kj/async.c++

```diff
@@ -878,6 +886,11 @@ struct FiberStack::StartRoutine {
     ptr |= static_cast<uintptr_t>(static_cast<uint>(arg2)) << (sizeof(ptr) * 4);
     auto stack = reinterpret_cast<FiberStack*>(ptr);
 
+    /// Jump back into the fiber stack constructor
```

Comment:
nitpick: three slashes?

Reply:
The Rustdoc Way:tm: and a force of habit :)